app/core/admin.py

```diff
@@ -2,7 +2,7 @@
 
 from core.models import (CourseDetailHighlight, CourseSpotlight, SearchFilter,
                          SearchSortOption, XDSConfiguration,
-                         XDSUIConfiguration)
+                         XDSUIConfiguration, CourseInformationMapping)
```

Comment:
Just wondering, does order matter here at all?  My OCD thinks these should be alphabetized, but maybe that doesn't make sense to do.

Reply:
@rglasgow-deloitte we use isort for this.
Inside your app root folder (make sure you're inside the virtual env): run pip install isort
then once it's installed you can: cd into the app directory -> run "isort ." 
this should sort all the imports for you

Reply:
This has been fixed in all files.